src/main/resources/application.yaml

```diff
@@ -3,6 +3,10 @@
 
 stargate:
 
+  database:
+    limits:
+      max-collections: 5
```

Comment:
Not 100% required since `DatabaseLimitsConfig` has defaults, but shown as documentation